src/serverconf/src/main/resources/serverconf.hbm.xml

```diff
@@ -101,7 +101,8 @@
         </id>
 
         <many-to-one name="serviceDescription" access="field" column="servicedescription_id"
-                     class="ee.ria.xroad.common.conf.serverconf.model.ServiceDescriptionType" cascade="all"/>
+                     class="ee.ria.xroad.common.conf.serverconf.model.ServiceDescriptionType"
+                     cascade="none"/>
```

Comment:
Cascade all->none, does something break?

Reply:
Had a shared team discussion about this; it is almost certain that "none" is more correct than "all". Having it "all" would break some implementations in rest-ui-api module. Value was changed in https://jira.niis.org/browse/XRDDEV-557

Change to "none" could break some implementations that would create both service and serviceDescription and then persist both by persisting the service. This would probably only happen in tests. We have run all the tests we have against this configuration, so it is quite likely this works. We can still study it more in a separate ticket, https://jira.niis.org/browse/XRDDEV-723?focusedCommentId=12716&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-12716